src/app_service/service/contacts/service.nim

```diff
@@ -363,7 +363,6 @@ QtObject:
       isCurrentUser = false,
     )
     self.addContact(contact)
-    self.events.emit(SIGNAL_CONTACT_UPDATED, ContactArgs(contactId: pubkey))
```

Comment:
This was the source of the slow down. It was used to populate the contact model with unknown contacts so that the profile popup would work, but it just ended up being called thousands of time.

We populate the contact model on demand instead